Sources/Overload/OvEditor/src/OvEditor/Core/EditorActions.cpp

```diff
@@ -292,7 +300,7 @@ void OvEditor::Core::EditorActions::BuildAtLocation(const std::string & p_config
 				}
 				else
 				{
-					OVLOG_ERROR("Builder folder for \"" + p_configuration + "\" not found. (Missing \"" + builderFolder + "\")");
+					OVLOG_ERROR("Builder folder for \"" + p_configuration + "\" not found. Verify you have compiled Engine source code in 'Release' configuration.");
```

Comment:
What if the editor has been compiled in Release mode only, and the user tries to generate a Development mode build? It looks to me like the user will get prompted an error message concerning the Release configuration not being compiled while in this case it should specify that the Development configuration hasn't been compiled

Reply:
Requested changes made. Let me know if any further adjustments are required.

Reply:
LGTM!